src/advanced/node-management.md

```diff
@@ -559,258 +556,193 @@ Response is a JSON object with the following fields:
-          "schema_version": 0,
-          "state_hash": "8ad43ece286a45b6269183cc3fab215066c31054b06c9ef391697b88c03bb63c",
-          "tx_count": 1000,
-          "tx_hash": "687d757b742a28110c40d9246586767a49b4c2521732b9ac23686481168d7e6c"
-        },
-        {
-          "height": "6",
-          "prev_hash": "f9ecccb5be51363744cffbb0b8241c2541c3aa7e53f1b9cca01d49d4991ae693",
-          "proposer_id": 3,
-          "schema_version": 0,
-          "state_hash": "8ad43ece286a45b6269183cc3fab215066c31054b06c9ef391697b88c03bb63c",
-          "tx_count": 1000,
-          "tx_hash": "577b4d67d045d8aa08e1b63a08fe77f09ac2ae8b2f46a6c3294e4ef4efaceb6c"
-        },
-        {
-          "height": "2",
-          "prev_hash": "d8d583444b823db312be85dc5dfa6d41b658c2bfe8caf97cb4b7372f2154ad4b",
-          "proposer_id": 3,
-          "schema_version": 0,
-          "state_hash": "8ad43ece286a45b6269183cc3fab215066c31054b06c9ef391697b88c03bb63c",
-          "tx_count": 1000,
-          "tx_hash": "94f251c0350c95024f46d26cbe0f9d2ea309e2817da4bab575fc4c571140291f"
-        }
-      ],
-      "range": {
-        "end": 101,
-        "start": 2
-      }
+    "range": {
+      "start": 6,
+      "end": 288
+    },
+    "blocks": [{
+      "proposer_id": 3,
+      "height": 26,
+      "tx_count": 1,
+      "prev_hash": "932470a22d37a5a995519e01c50eab7db9e0e978f5b17f8342030ae3f066af82",
+      "tx_hash": "5cc41a2a7cf7c0d3a15ab6ca775b601208dba7d506e2f27368702b3334d37583",
+      "state_hash": "4d7bb34d7913e0784c24a1e440532e72900eb380129a54dbaac6ad9286f9d567"
+    },{
+      "proposer_id": 2,
+      "height": 21,
+      "tx_count": 1,
+      "prev_hash": "aa4ec89740a4ec380e8bcab0aedd0f5449184eb33b65ede5bb67e5e55e2dd004",
+      "tx_hash": "dcb05a3bd61f9b637335472802d8ab6026c8486dae3b4062ce48d561949c49af",
+      "state_hash": "e4ea2c6118326c6b00cd14ec7b8fb4cbf198eb4e65149ef3a96761740fc516c6"
+    },{
+      "proposer_id": 1,
+      "height": 16,
+      "tx_count": 1,
+      "prev_hash": "7183517c34e94ecc10a3e13269da2bfadb6e87eea86453a1946ebdfa9c4dae83",
+      "tx_hash": "362bc50ed56d33944a0d33fbac2a25fc08ceb8dc1aced1f38147b3da3d022bc1",
+      "state_hash": "00cca5682b677d4b4ac644d2ddae09ca5e260fb67c735df22774f2e983d24ef5"
+    },{
+      "proposer_id": 0,
+      "height": 11,
+      "tx_count": 1,
+      "prev_hash": "9297ef66d1d9ec286c00aec779f2dc273b3371e792bbc9c6635d00f9c4a6fa80",
+      "tx_hash": "c7aa20695380846e3f274d3d51c68e864e66e46f2618aa5fbd55d597675b9e6a",
+      "state_hash": "deb57ff0f82c9d2514dc51785675544e27b3054512ea62dce2c8e30ce6d91e77"
+    },{
+      "proposer_id": 3,
+      "height": 6,
+      "tx_count": 1,
+      "prev_hash": "dbec8f64a85ab56985c7ab7e63a191764f4d5c373c677f719c2f9ddf13b9d5a1",
+      "tx_hash": "ffee3d630f137aecff95aece36cfe4dc1b42f688d474219cb30d44c85cf36b1f",
+      "state_hash": "8ac9f2af6266b8e9b61fa7f3fcdd170375fb1bf8cc8d474904abe3672b44906e"
+    }],
+    "times": ["2019-02-21T13:01:44.321051Z",
```

Comment:
Nit: weird formatting of the array. I believe the following format would work better:

```json
{
  "times": [
    "2019-02-21T13:01:44.321051Z",
    "2019-02-21T13:01:43.287648Z"
  ]
}
```